back/node_watcher/package.json

```diff
@@ -24,6 +24,7 @@
     "start": "ts-node ./src/index.ts"
   },
   "dependencies": {
+    "@polkadot/api": "^1.0.0-beta.18",
```

Comment:
Thibaut removed it from this package, so that we only put it in the root one

Reply:
yeah but when we dockerize it, the image will have no `@polkadot/api`

Reply:
ah right